Metadata.Rmd

```diff
@@ -0,0 +1,660 @@
+
+The `Config/Needs/*` convention is handy because it allows a developer to use `DESCRIPTION` as their definitive record of package dependencies, while maintaining a clean distinction between true runtime dependencies versus those that are only needed for specialized development tasks.
+
+<!-- re: describing different types of dependencies, another term you see for "runtime" dependency is "production" -->
+
+## Title and description: What does your package do? {#description-title-description}
+
+The title and description fields describe what the package does.
+They differ only in length:
+
+* `Title` is a one line description of the package, and is often shown in a
+  package listing.
+  It should be plain text (no markup), capitalised like a title, and NOT end in
+  a period.
+  Keep it short: listings will often truncate the title to 65 characters.
+* `Description` is more detailed than the title.
+  You can use multiple sentences, but you are limited to one paragraph.
+  If your description spans multiple lines (and it should!), each line must be
+  no more than 80 characters wide.
+  Indent subsequent lines with 4 spaces.
+
+The `Title` and `Description` for ggplot2 are:
+
+```yaml
+Title: Create Elegant Data Visualisations Using the Grammar of Graphics
+Description: A system for 'declaratively' creating graphics,
+    based on "The Grammar of Graphics". You provide the data, tell 'ggplot2'
+    how to map variables to aesthetics, what graphical primitives to use,
+    and it takes care of the details.
+```
+
+A good title and description are important, especially if you plan to release your package to CRAN, because they appear on the CRAN download page as follows:
+
+<!-- TODO: I know my hacky diagram might need replacement, but at least it's more up-to-date. -->
+
+```{r cran-package-page}
+#| echo = FALSE, out.width = "100%",
+#| fig.cap = "The CRAN page for ggplot2, highlighting Title and Description."
+knitr::include_graphics("diagrams/cran-package-ggplot2.png")
+```
+
+You'll notice that `Description` only gives you a small amount of space to describe what your package does.
+This is why it's so important to also include a `README.md` file that goes into much more depth and shows a few examples.
+You'll learn about that in [README.md](#readme).
+
+## Author: who are you? {#description-authors}
+
+To identify the package's author, and whom to contact if something goes wrong, use the `Authors@R` field.
+This field is unusual because it contains executable R code rather than plain text. Here's an example:
+
+```yaml
+Authors@R: person("Hadley", "Wickham", email = "hadley@rstudio.com",
+  role = c("aut", "cre"))
+```
+
+```{r}
+person("Hadley", "Wickham", email = "hadley@rstudio.com", 
+  role = c("aut", "cre"))
+```
+
+This command says that both the author (aut) and the maintainer (cre) is Hadley Wickham, and that his email address is `hadley@rstudio.com`.
```

Comment:
I think `cre` needs a footnote; despite being short for creator it's used for the current maintainer, not the initial creator of the package